### DISCUSSION TITLE
unable to start Firefly

### DISCUSSION THREAD

**Discussion by CryptoMaXeY** [iotaledger/firefly#7336] [Help & Support]
App Version: 1.7.3

platform: macOS
platformVersion: 11.7.9
platformArchitecture: x64
cpuCount: 8
totalMem: 16384.0 MB
freeMem: 48.9 MB

Error Type: [Preload Context] Error

Error: node-loader:
Error: dlopen(/private/var/folders/j9/my4_p1y51_j8d3s0bpcrhmqm0000gn/T/AppTranslocation/6841C93C-C3AD-4032-B838-EAA18352A0BC/d/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node, 1): Symbol not found: __ZNSt3__115basic_stringbufIcNS_11char_traitsIcEENS_9allocatorIcEEE7seekoffExNS_8ios_base7seekdirEj
  Referenced from: /private/var/folders/j9/my4_p1y51_j8d3s0bpcrhmqm0000gn/T/AppTranslocation/6841C93C-C3AD-4032-B838-EAA18352A0BC/d/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node (which was built for Mac OS X 13.0)
  Expected in: /usr/lib/libc++.1.dylib
 in /private/var/folders/j9/my4_p1y51_j8d3s0bpcrhmqm0000gn/T/AppTranslocation/6841C93C-C3AD-4032-B838-EAA18352A0BC/d/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node

**Comment by begonaalvarezd** [comment]
Duplicate of https://github.com/iotaledger/firefly/issues/7324 we found the issue and will release an update very soon. Apologies for the troubles 🙏🏼
